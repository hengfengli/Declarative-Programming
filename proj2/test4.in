+---+---+---+
|...|..5|..2|
|...|...|.13|
|13.|.8.|5..|
+---+---+---+
|..8|..9|2..|
|...|648|...|
|..9|2..|4..|
+---+---+---+
|..2|.6.|.45|
|68.|...|...|
|7..|5..|...|
+---+---+---+
