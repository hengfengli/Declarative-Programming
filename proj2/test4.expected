+---+---+---+
|894|315|672|
|257|496|813|
|136|782|594|
+---+---+---+
|478|159|236|
|321|648|759|
|569|273|481|
+---+---+---+
|912|867|345|
|685|934|127|
|743|521|968|
+---+---+---+
